run/send-push-message

```diff
@@ -51,7 +51,36 @@ function send_sns () {
   python3 /root/bin/send_sns.py -t $sns_topic_arn -s "$TITLE" -m "$MESSAGE"
 }
 
+function send_discord() {
+  log "Sending Discord message for moved files."
+
+  source /root/.teslaCamDiscordSettings
+
+  MSG_TIME=$(date --iso-8601=seconds)
+  MSG_USERNAME=${TITLE%?} # Remove ':' from title
+  AVATAR_URL='https://i.imgur.com/EuliK6R.png'
```

Comment:
This should probably be defined in the /root/.teslaCamDiscordSettings file, so the user can customize it.
What happens if the avatar url is not specified? My concern is that this is a random URL on an external website and there's no guarantee it will remain valid or point at the same image in the future.
I think it would be better if this image was hosted as part of the teslausb project, but I don't know what github's policy on hosting images is.

Reply:
I agree with you, but MSG_TIME must remain static as --iso-8601 is the only supported ISO format by Discord API.  

The AVATAR_URL variable can be empty but has to exist and it will still work. I dont think Imgur ever deletes anything with out a complaint so I feel Imgur is a pretty reliable source as I uploaded the picture. I suggest that we add AVATAR_URL to both teslaCamDiscordSettings and teslausb_setup_variables.conf.sample so users can use what ever image URL they want. I will also add MSG_USERNAME to teslaCamDiscordSettings.